tests/rptest/services/redpanda.py

```diff
@@ -3698,15 +3698,25 @@ def _get_object_storage_report(
         environment = ' '.join(f'{k}=\"{v}\"' for k, v in vars.items())
         output = node.account.ssh_output(
             f"{environment} rp-storage-tool --backend {backend} scan-metadata --source {bucket}",
-            combine_stderr=False,
+            combine_stderr=True,
```

Comment:
Not necessary for this PR, but at some point it would be good to fix ducktape itself to enable capturing stderr and stdout neatly (there's already a redpanda branch that we use)

Reply:
I saw some inconsistencies in the ci, so I just implemented _ssh_output_stderr, mostly copying the one from kerberos.py. if it's good I can later upstream it and remove it from redpanda.py